src/MonologCreator/Processor/ExtraFieldProcessor.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+
+namespace MonologCreator\Processor;
+
+/**
+ * Class ExtraFieldProcessor
+ *
+ * Allows adding additional high-level or special fields to the log output.
+ *
+ * @package MonologCreator\Processor
+ * @author Sebastian Götze <s.goetze@bigpoint.net>
+ */
+class ExtraFieldProcessor implements \Monolog\Processor\ProcessorInterface
+{
+    /**
+     * Array to hold additional fields
+     *
+     * @var array
+     */
+    protected $_extraFields = array();
+
+    /**
+     * ExtraFieldProcessor constructor.
+     *
+     * @param array|null $extraFields
```

Comment:
Why is there a `null` in the parameter definition here? According to the actual code below it can't be null as you define it as type `array`.

Also the entire doc block is unneeded, as the type hinting at the constructor itself is enough for IDEs to work with.

Reply:
Forgot updating the doc after changing the default type. Will remove the doc block.